libclamav/dsig.c

```diff
@@ -256,13 +288,18 @@ int cli_versig(const char *md5, const char *dsig)
     if (strncmp(md5, pt2, 32)) {
         cli_dbgmsg("cli_versig: Signature doesn't match.\n");
         free(pt2);
-        return CL_EVERIFY;
+        ret = CL_EVERIFY;
+        goto out;
     }
 
     free(pt2);
 
     cli_dbgmsg("cli_versig: Digital signature is correct.\n");
-    return CL_SUCCESS;
+    ret = CL_SUCCESS;
+out:
```

Comment:
Same comment regarding `out` --> `done`, for consistency with rest of code base.

Reply:
done.